ambry-store/src/main/java/com/github/ambry/store/BlobStore.java

```diff
@@ -87,7 +87,8 @@
   private final RemoteTokenTracker remoteTokenTracker;
   private final AtomicInteger errorCount;
   private final AccountService accountService;
-
+  //The max replication lag indicating how far local store is behind peers.
+  private long localStoreMaxLagFromPeer;
```

Comment:
make this `volatile`, as it will be updated and accessed by different threads

Reply:
Thx for catching up, updated.